pkg/network/tracer/tracer_test.go

```diff
@@ -1564,7 +1564,8 @@ func TestHTTPSViaOpenSSLIntegration(t *testing.T) {
 
 	// Spin-up HTTPS server
 	serverDoneFn := testutil.HTTPServer(t, "127.0.0.1:443", testutil.Options{
-		EnableTLS: true,
+		EnableTLS:        true,
+		EnableKeepAlives: true,
```

Comment:
can you actually run the same test with keep-alive turned on and off? sort of similarly to what we do in `monitor_test.go`